tmcprototype/configure_hdbpp.py

```diff
@@ -0,0 +1,201 @@
+#         print("SetAttributeName: ", conf_manager_proxy.SetAttributeName)
+#
+#         # SetArchiver
+#         conf_manager_proxy.write_attribute("SetArchiver",
+#                                            "tango://archiver-1234-databaseds:10000/archiving/hdbpp/eventsubscriber01")
+#         print("SetArchiver: ", conf_manager_proxy.SetArchiver)
+#
+#         # SetStrategy
+#         conf_manager_proxy.write_attribute("SetStrategy", "ALWAYS")
+#         print("SetStrategy: ", conf_manager_proxy.SetStrategy)
+#
+#         # SetPollingPeriod
+#         conf_manager_proxy.write_attribute("SetPollingPeriod", 1000)
+#         print("SetPollingPeriod: ", conf_manager_proxy.SetPollingPeriod)
+#
+#         # SetEventPeriod
+#         conf_manager_proxy.write_attribute("SetPeriodEvent", 3000)
+#         print("SetPeriodEvent: ", conf_manager_proxy.SetPeriodEvent)
+#         ret_value = True
+#
+#     except Exception as except_occured:
+#         print("except_occured: ", except_occured)
+#         ret_value = False
+#     return ret_value
+
+def cm_add_attribute():
+    ret_value = False
+    try:
+        # Add Attribute
+        conf_manager_proxy.command_inout("AttributeAdd")
+        ret_value = True
+    except DevFailed as df:
+        # print("cm_add_attribute except_occured: ", df)
+        str_df = str(df)
+        print("cm_add_attribute except_occured reason: ", str_df)
+
+        if "reason = Already archived" in str_df:
+            start_archiving()
+            ret_value = True
+        else:
+            ret_value = False
+    return ret_value
+
+def start_archiving():
+    try:
+        conf_manager_proxy.command_inout("AttributeStart", "sys/tg_test/1/ampli")
+    except Exception as except_occured:
+        print("start_archiving except_occured: ", except_occured)
+
+# Main entrypoint of the script.
+if __name__ == "__main__":
+    conf_manager_device_fqdn = ""
+    evt_subscriber_device_fqdn = ""
+    attr_list_file = ""
+
+    ## parse arguments
+    try:
+        opts, args = getopt.getopt(sys.argv[1:], "c:e:a:", ["cm=", "es=", "attrfile="])
+    except getopt.GetoptError:
+        print("Please provide proper arguments.")
+        print("Usage: $python configure_hdbpp.py --cm=<FQDN> --es=<FQDN> --attrfile=<filepath> OR")
+        print("       $python configure_hdbpp.py -cm <FQDN> -e <FQDN> -a <filepath>")
+        print("       cm: FQDN of HDB++ Configuration Manager")
+        print("       es: FQDN of HDB++ Event subscriber")
+        print("       infile: File containing FQDNs of attributes to archive")
+        sys.exit(2)
+
+    for opt, arg in opts:
+        if opt in ("-c", "--cm"):
+            conf_manager_device_fqdn = arg
+        elif opt in ("-e", "--es"):
+            evt_subscriber_device_fqdn = arg
+        elif  opt in ("-a", "--attrfile"):
+            attr_list_file = arg
+
+    print("conf_manager_device_fqdn: ", conf_manager_device_fqdn)
```

Comment:
Remove print statements if not required. Also check for other instances in the script.

Reply:
Removed unnecessary print statements.

Reply:
Commit id is: 6abd5d877d834e9322fd4eef40b4eb93f2a010fb